README.md

```diff
@@ -189,7 +210,7 @@ Read those from the command line by editing the part where the arguments are rea
 
     let [, , sender, receiver, host, port] = process.argv;
     host = host || 'localhost';
-    port = port || 7600;
+    port = port || 6865;
     if (!sender || !receiver) {
         console.log('Missing sender and/or receiver arguments, exiting.');
         process.exit(-1);
```

Comment:
Please add the full invocation with sender / receiving below as you do further down, e.g. `run start Alice Bob`

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/05ee6db892c9d4f5979aa06a323c3810a22ff091